core/src/main/java/hudson/model/AsyncAperiodicWork.java

```diff
@@ -80,8 +127,8 @@ public void run() {
                         l.closeQuietly();
                     }
 
-                    logger.log(Level.INFO, "Finished "+name+". "+
-                        (System.currentTimeMillis()-startTime)+" ms");
+                    logger.log(getNormalLoggingLevel(), "Finished {0}. {1,number} ms",
+                            new Object[]{name, System.currentTimeMillis() - startTime});
```

Comment:
yes but I had issues getting the stopped time and ensuring that the stop time would always be populated... I have thought some more and now have a code path that works